react-graphql-apollo-tutorial.md

```diff
@@ -371,45 +371,45 @@ It's up to you to come up with an own folder/file structure for your components
 * registerServiceWorker.js
 * style.css
 
-As you can see, the folders mostly represent React components. Whereas some components will be reusable UI components such as the Input and Link components, other components such as Repository and Profile components are domain specific for the GitHub client application. In addition, only the top level folders are specified for now. You will introduce further folders and files later on your own. Moreover the constants folder has only one file to specify the routes of the application which will be introduced later as well. After all, you may want to navigate from a page which shows repositories of an organization (Organization component) to a page which shows repositories of yourself (Profile component).
+The folders primarily represent React components. Some components will be reusable UI components such as the Input and Link components, while other components like Repository and Profile components are domain specific for the GitHub client application. Only the top level folders are specified for now, though more can be introduced later if you choose. Moreover, the 'constants' folder has only one file to specify the application's routes, which will be introduced later. You may want to navigate from a page that shows repositories of an organization (Organization component) to a page which shows repositories of yourself (Profile component).
```

Comment:
"Moreover, the 'constants' folder" to "Moreover, the *constants* folder" <- I tend to put file or folder names as italics.